### ISSUE TITLE
Throw Exception for 1408, 1412, 1416, etc

### ISSUE THREAD

**Issue by fardinfrd** [samanzamani/PersianDate#35]
Hi developers
I have problem with this code:

```
PersianDate persianDate = new PersianDate();
persianDate.setShYear(1408);
persianDate.setShMonth(1);
persianDate.setShDay(1);

//some lines of code

new PersianDate(persianDate.getTime());
```

Upper code throws exception:
Method threw 'java.lang.ArrayIndexOutOfBoundsException' exception.
java.lang.ArrayIndexOutOfBoundsException: length=13; index=-1

I use version 0.8.

thanks

**Comment by samanzamani** [comment]
Hi @fardinfrd 
Thank you for the report
I will check this

**Comment by samanzamani** [comment]
Bug fixed in v1.0
